Tests/LibJS/test-value-js.cpp

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2022, David Tuin <davidot@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibJS/Runtime/Value.h>
+#include <LibTest/TestCase.h>
+
+using namespace JS;
```

Comment:
What's this needed for?

Reply:
So that I can use the name of the type in the name of the test in the `TEST_NULLPTR_INPUT` macro.

But if you don't like it I can remove it

Reply:
it's fine, just wasn't obvious to me :)